OptimizelySDK/Optimizely.cs

```diff
@@ -470,17 +470,25 @@ public virtual bool IsFeatureEnabled(string featureKey, string userId, UserAttri
             bool featureEnabled = false;
             var sourceInfo = new Dictionary<string, string>();
             var decision = DecisionService.GetVariationForFeature(featureFlag, userId, config, userAttributes);
+            var variation = decision.Variation;
+            var decisionSource = FeatureDecision.DECISION_SOURCE_ROLLOUT;
+            if (decisionSource != null)
```

Comment:
when this will be false?  Looks like you are trying to say 
`decision.Source != null`

I will recommend to rewrite like this. 
`var decisionSource = decision?.Source ?? FeatureDecision.DECISION_SOURCE_ROLLOUT;`